src/aiohabit/outputs/utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for output modules."""
+
+import socket
+from socket import error as socket_error
+
+
+def resolve_hostname(ip):
+    """Resolve IP address to hostname."""
+    try:
+        return socket.gethostbyaddr(ip)[0]
+    except socket_error:
+        return None
+
+
+def get_val(values):
```

Comment:
This is called mostly passing a list of only 2 items, so could be replaced with: ```result = first or second```.
But, if you think this is valuable I would rename it to something like `get_first_valid`.

Reply:
Renamed. I mainly implemented it for the case with 3 items. But after the rename and push, I see that it is used a lot for the 2 value cases where it indeed can use the 'or'.

Reply:
So after second, more careful read. I've rewrote it as suggested to the 'or' form and in the end removed the method completely. And the code looks better.